monkey/infection_monkey/system_info/windows_cred_collector/windows_cred_collector.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from typing import List
+
+from infection_monkey.system_info.windows_cred_collector.pypykatz_handler import get_windows_creds
+from infection_monkey.system_info.windows_cred_collector.windows_credential import WindowsCredential
+
+LOG = logging.getLogger(__name__)
+
+
+class WindowsCredentialCollector(object):
+
+    @staticmethod
+    def get_creds():
+        creds = get_windows_creds()
+        return WindowsCredentialCollector.cred_list_to_cred_dict(creds)
+
+    @staticmethod
+    def cred_list_to_cred_dict(creds: List[WindowsCredential]):
+        cred_dict = {}
+        for cred in creds:
+            # Lets not use "." and "$" in keys, because it will confuse mongo.
```

Comment:
Is this still true? 

>  starting in MongoDB 3.6, the server permits storage of field names that contain dots (i.e. .) and dollar signs (i.e. $)

From [Mongo reference](https://docs.mongodb.com/manual/reference/limits/#Restrictions-on-Field-Names)

Reply:
We're not working with mongoDB though https://stackoverflow.com/questions/12397118/mongodb-dot-in-key-name

Reply:
For pymongo - yes
![image](https://user-images.githubusercontent.com/36815064/84012505-54182d00-a980-11ea-9575-174a62a59d01.png)

Reply:
Doesn't matter, it's just a key. we use the value of 'username' field for the actual brute force.